dor/providers/file_system_file_provider.py

```diff
@@ -14,6 +15,21 @@ def get_descriptor_dir(self, file_path: Path) -> Path:
         return file_path.parent
 
     def get_norm_path(self, base_path: Path, path_to_apply: str) -> Path:
-        combined_path = os.path.join(base_path, path_to_apply)
+        combined_path = self.get_combined_path(base_path, path_to_apply)
         resolved_combined_path = Path(os.path.normpath(combined_path))
         return resolved_combined_path
+
+    def get_combined_path(self, base_path: Path, path_to_apply: str) -> Path:
+        return Path(os.path.join(base_path, path_to_apply))
```

Comment:
I think you can just return `base_path / path_to_apply` without `Path` and `os.path.join`